sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/implementation/DocumentServiceLeaseStore.java

```diff
@@ -143,10 +152,11 @@ public DocumentServiceLeaseStore(
                 if (throwable instanceof CosmosClientException) {
                     CosmosClientException e = (CosmosClientException) throwable;
                     if (e.statusCode() == ChangeFeedHelper.HTTP_STATUS_CODE_CONFLICT) {
+                        logger.info("Lease synchronization document was acquired by a different instance");
                         return Mono.just(false);
                     }
                 }
-
+                logger.warn("Unexpected exception thrown", throwable);
```

Comment:
Should be debug. Warn is too much of noise.

Reply:
Fixed